src/common/icons/cross-icon.tsx

```diff
@@ -6,6 +6,25 @@ import { NamedSFC } from '../react/named-sfc';
 
 export const CrossIcon = NamedSFC('CrossIcon', () => (
     <span className="check-container">
-        <span className="cross-icon" />
```

Comment:
Since we're using SVG instead of CSS to create the cross icon, can we remove the CSS rules for `cross-icon`?